topics/08_Known-Issues/01_Known_Issues_and_Limitations_2016_Volume_2.md

```diff
@@ -391,6 +391,22 @@ When you use the minValue and maxValue options in the igEditors, you should be a
 ![](../images/images/negative.png)
 			</td>
         </tr>
+		
+		<tr>
+            <td>
+Spinning up when the maximum value is reached and spinWrapAround option is true is not possible if  maxValue is not set with dataMode being float, long or double
```

Comment:
Suggestion: "Spin wrap around is not possible after reaching a default limit value provided by numeric data modes float, long or double"
(as this is true if you hit the default min values as well)